controllers/TasksController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace app\controllers;
+
+use app\exceptions\TaskException;
+use app\models\Tasks;
+use yii\web\Controller;
+
+class TasksController extends Controller
+{
+    public $layout = 'inner.php';
```

Comment:
можно без расширения, фреймворк сам поймет, он достаточно умный =)

```suggestion
    public $layout = 'inner';
```